fiatconnect-api.md

```diff
@@ -597,9 +600,8 @@ receive by providing `fiatAmount` worth of the fiat currency. If `cryptoAmount`
 of fiat currency required in order to receive the requested amount of crypto.
 
 The `quote.fiatType`, `quote.cryptoType`, `quote.fiatAmount`, and `quote.cryptoAmount` fields in the response body MUST correspond to the query parameters provided to the endpoint.
-The `quote.guaranteedUntil` field represents the time that the quote is guaranteed until, as a UNIX timestamp. A server MAY choose to provide this value. If it does not, the client should assume that
-the server makes no guarantees about the actual conversion rate when performing a transfer for the given quote. If a server provides this value in the response, it MUST honor the
-provided conversion rate when initiating transfers until the time indiciated in the `quote.guaranteedUntil` field.
+The `quote.guaranteedUntil` field represents the time that the quote is guaranteed until, as a UNIX timestamp. A server MUST provide this value, and MUST honor the
```

Comment:
Can you add something in here explaining the `quoteId` field in the response? e.g. "The `quoteId` field in the response body is a globally unique identifier for the returned quote, that the client uses to initiate a transfer with a guaranteed currency conversion rate" or something to that effect.

Reply:
Also, it could be a good idea to explicitly require servers to maintain a history of quotes, even after their `guaranteedUntil` time. Without this, a server could throw out a `quoteId` that was past its `guaranteedUntil` time, rendering the `QuoteExpired` response code from the transfer endpoint useless.

Reply:
I'd like to avoid requiring providers to maintain a history of expired quotes, so I updated the errors to account for this a bit better

Reply:
It's a good point that a QuoteExpired error doesn't mean much if there's no instructions on how long to keep the quotes for